fix underscores in tests (#159)
mate-academy/py-taxi-service-authentication: README.md

![image](https://mate-academy-images.s3.eu-central-1.amazonaws.com/python_pr_with_images.png)

> **Note:** the `-` hyphen usage in URLs names is the best practice in Django, whilst the `_` underscore is the best practice in DRF.
> 

Reviewer: ```suggestion
**Note:** the `-` hyphen we use in URLs names in our Django course, whilst the `_` underscore we use in our DRF course.

```
